target/linux/ath79/image/generic.mk

```diff
@@ -305,6 +305,16 @@ define Device/aruba_ap-105
 endef
 TARGET_DEVICES += aruba_ap-105
 
+define Device/atheros_db120
+  SOC := ar9344
+  DEVICE_VENDOR := Atheros
+  DEVICE_MODEL := DB120
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-storage
+  IMAGE_SIZE := 7808k
+  SUPPORTED_DEVICES += db120
```

Comment:
I didn't find any support in ar71xx. I assume SUPPORTED_DEVICES can be dropped?

Reply:
It used to be supported in ar71xx, but happy to drop this anyway.

Reply:
Ah, didn't find it because it is in legacy.mk. If sysupgrade from ar71xx works in principle, you can keep the line.